proto/mixer.proto

```diff
@@ -595,17 +595,40 @@ message GetStatValueRequest {
   string place = 1;
   // dcid of the stat var.
   string stat_var = 2;
-  // (optional) date of the stat.
+  // (optional) date of the stat. The latest date will be used if unspecified.
   string date = 3;
 }
 
 message GetStatValueResponse {
   double value = 1;
 }
 
+// Request for GetStatSeries service.
+message GetStatSeriesRequest {
+  // dcid of the place.
+  string place = 1;
+  // dcid of the stat var.
+  string stat_var = 2;
+  // (optional) data provenance domain like "cdc.org".
+  string provenance_domain = 3;
```

Comment:
If the general goal is to allow as filter extra disambiguating properties for a series within a StatVar, then provenance_domain is not one of them. It is not exactly a property of StatVarObs/Obs.

Reply:
removed for now